build-system/compile/internal-version.js

```diff
@@ -15,12 +15,22 @@
  */
 'use strict';
 
-const argv = require('minimist')(process.argv.slice(2));
+const minimist = require('minimist');
 const {gitCommitFormattedTime} = require('../common/git');
 
+// Allow leading zeros in --version_override, e.g. 0000000000001
+const argv = minimist(process.argv.slice(2), {
+  string: ['version_override'],
```

Comment:
I think this is unnecessary. `minimist` should parse `--version_override whatever` as `version_override: 'whatever'` by default.

Reply:
It is necessary. I only added it after testing.